syntax = "proto2";

package chrome_browser_media;

// Chrome requires this.
option optimize_for = LITE_RUNTIME;

// Next available id: 4
message PreloadedData {
  // The huffman tree.
  repeated uint32 huffman_tree = 1;

  // The trie data.
  repeated uint32 trie_data = 2;

  // The size of the trie in bits.
  optional sfixed32 trie_bits = 3;

  // The root position of the trie.
  optional sfixed32 root_position = 4;
}
